src/client.ts

```diff
@@ -14,9 +15,10 @@ async function sendInformation() {
       },
     },
     (response) => {
-      console.table(response.statusCode); // 200
+      console.table(response.statusCode);
     },
   );
+  await update(data, { credentials: { password: password } });
```

Comment:
Aunque esto funciona, creo que no es necesario hacerlo una promesa, ya que es una llamada síncrona muy veloz y sin hacer un request por el que haya que esperar. 

```suggestion
  data.credentials.password = password; 
```

Si quisieras usar tu método, podría simplificarse así: 

```suggestion
  await update(data, { credentials: { password} });
```

También puedes usar esto: 

```
data = { ...data, credentials: { password } }
```

Reply:
Muchas gracias por las observaciones, son de mucha ayuda, creo que yo mismo dificulte las cosas. Ya he corregido esto y los cambios han sido subidos.